source/PreferencesPanel.cpp

```diff
@@ -201,6 +202,10 @@ bool PreferencesPanel::Click(int x, int y, int clicks)
 					speed = 20;
 				Preferences::SetScrollSpeed(speed);
 			}
+			else if (zone.Value() == DATE_FORMAT)
+			{
```

Comment:
Don't need the braces here.

Reply:
Resolved by c11da73fa475e4307f0e3dfd838c5cf1c95a45b1